eng/build.ps1

```diff
@@ -59,16 +59,8 @@ if ($dailytest -or $privatebuildpath -ne "") {
 
 # Remove the private build registry keys
 if ($cleanupprivatebuild) {
-    $dotnetinstallroot = Join-Path $reporoot ".dotnet"
-    if ($architecture -eq "x86") {
-        $dotnetinstallroot = Join-Path $dotnetinstallroot "x86"
-    }
-    if (Test-Path $dotnetinstallroot\RemovePrivateTesting.reg) {
-        regedit $dotnetinstallroot\RemovePrivateTesting.reg
-        # delete the add reg file so registry is edited again
-        del $dotnetinstallroot\AddPrivateTesting.reg
-    }
-    exit 0
+    Invoke-Expression "& `"$engroot\common\msbuild.ps1`" $engroot\CleanupPrivateBuild.csproj /v:$verbosity /t:CleanupPrivateBuild /p:BuildArch=$architecture /p:TestArchitectures=$architecture"
```

Comment:
why the additional project? Maybe just keep the one file and call the target as we have here?

Reply:
It needs to be a csproj with the SDK imported to work. Just building the proj file didn't work.